ui/app/manager/src/pages/page-assets.ts

```diff
@@ -343,4 +345,11 @@ export class PageAssets extends Page<AssetsStateKeyed>  {
             callHandler: !silent
         });
     }
+
+    protected _onLoadUserEvent(event: OrAssetViewerLoadUserEvent, silent: boolean = false) {
```

Comment:
Just checking with @richturner here.

I don't know how we should practice page switching.
Especially since this event is navigating to `users` (so outside of assets context)
Other than in `or-map-asset-card.ts` / `page-map.ts` we do not practice it anywhere,
so just checking whether the practice is okay.

This is what @richturner implemented in 2020 in `page-map.ts` navigating to an individual asset:
```
protected onLoadAssetEvent(loadAssetEvent: OrMapAssetCardLoadAssetEvent) {
    router.navigate(getAssetsRoute(false, loadAssetEvent.detail));
}
```